packages/pose-core/src/factories/setter.ts

```diff
@@ -97,6 +97,19 @@ const resolveTransition = <V, A>(
   if (typeof transition === 'function') {
     resolvedTransition = transition(props);
 
+    /**
+     * transition: () => { d: () => tweenFn }
+     */
+    if (typeof resolvedTransition === 'object') {
```

Comment:
according to types I think it cant be non-object, so checking it here is a little bit pointless

Reply:
also when we are talking about types - this change has to be reflected in the typings